src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, createStore } from 'redux';
+import bookReducer from './books/books';
+import categoriesReducer from './categories/categories';
```

Comment:
- [ ] [OPTIONAL] Please consider installing the `redux-logger` npm package and importing `applyMiddleware` to use it. It will help you a lot in the development process 👍